tools/test-proxy/Azure.Sdk.Tools.TestProxy/RecordingHandler.cs

```diff
@@ -194,9 +194,10 @@ public async Task HandleRecordRequestAsync(string recordingId, HttpRequest incom
                 throw new HttpException(HttpStatusCode.BadRequest, $"There is no active recording session under id {recordingId}.");
             }
 
-            var entry = await CreateEntryAsync(incomingRequest).ConfigureAwait(false);
+            var entryTuple = await CreateEntryAsync(incomingRequest).ConfigureAwait(false);
```

Comment:
Can do something like this:
```suggestion
            (RecordEntry entry, byte[] bytes) = await CreateEntryAsync(incomingRequest).ConfigureAwait(false);
```